libtcmu_log.c

```diff
@@ -363,6 +528,21 @@ static struct log_buf *tcmu_log_initialize(void)
 	pthread_cond_init(&logbuf->cond, NULL);
 	pthread_mutex_init(&logbuf->lock, NULL);
 
+	darray_init(logbuf->outputs);
+
+	ret = create_syslog_output(TCMU_LOG_INFO, NULL);
+	if (ret < 0) {
+		fprintf(stderr, "create syslog output error \n");
+	}
+	ret = create_stdout_output(TCMU_LOG_DEBUG_SCSI_CMD);
```

Comment:
Did you change the behavior here? In the previous code normal debug and scsi debug would go to stdout. Was this intentional and why?